app/api.py

```diff
@@ -713,8 +713,8 @@ def post(self):
             self.info(
                 400,
                 message=
-                    'The given receiver_phone_number is not found in '
-                    'sms_number_to_repo config.',
+                    ('The given phone number: %s is not found in '
```

Comment:
I think this message could be made even less technical. How about "You've reached Person Finder, but there's not a repository assigned for %s." % receiver_phone_number
That way people would know the number's a Person Finder number and the message got to us.

Reply:
Done.